_topic_maps/_topic_map_rosa.yml

```diff
@@ -32,13 +32,13 @@ Topics:
   File: legal-notice
   Distros: openshift-rosa
 ---
-#Name: Release notes
-#Dir: rosa_release_notes
-#Distros: openshift-rosa
-#Topics:
-#- Name: Red Hat OpenShift Service on AWS release notes
-#  File: rosa-release-notes
-#---
+Name: Release notes
```

Comment:
The file, `id=`, directory, etc all say "release notes" but you have Named this "What's new". I suggest being consistent with naming.

Reply:
Fixed.